src/app/modules/server-settings/pages/manage-settings/manage-settings.component.ts

```diff
@@ -55,6 +55,21 @@ export class ManageSettingsComponent implements OnInit, OnDestroy {
                     return null;
                 }
             ]),
+            maxUploadSize: new FormControl('unlimited', [
```

Comment:
The key must match what the backend expects as well as the `formControlName` you set in the `manage-settings.component.html`. Currently there's a mismatch, which causes the build failure. 

![image](https://user-images.githubusercontent.com/2303663/97201620-52c69000-17bb-11eb-897f-e1addff6b42c.png)

You also need to add the same property in the `ServerSettings` under `server-settings.module.ts` for better IDE autocomplete support.

Reply:
One more thing - please follow the indentation in the file. We're using 4 spaces across the entire project. :)